widget/embedded/src/components/SwapDetails/SwapDetails.tsx

```diff
@@ -289,6 +292,17 @@ export function SwapDetails(props: SwapDetailsProps) {
                 onClick={handleCopy.bind(null, requestId || '')}>
                 <CopyIcon size={16} color="gray" />
               </IconButton>
+              <IconButton
+                variant="ghost"
+                style={{ padding: '0' }}
+                onClick={() =>
+                  window.open(
+                    `https://scan.rango.exchange/swap/${requestId}`,
+                    '_blank'
+                  )
```

Comment:
I think it's not a good idea to use window.open for the links. For example, user will not be able to copy the link anymore using the browser.